technical-reports/resolver/resolution-aliasing.md

```diff
@@ -0,0 +1,155 @@
+# Resolution Aliasing
+
+Aliasing allows for dynamic namespacing or renaming of token paths during resolution. This is particularly useful when integrating external token sets or avoiding naming conflicts.
+
+<aside class="issue">
+
+**Namespace vs Alias Terminology:** Should the `alias` property be renamed to `namespace` to avoid confusion with token aliases (references to other tokens)?
+
+**Redundancy with Modifier Names:** The `meta.alias` property may be redundant since modifiers already have a `name` property that could serve the same namespacing purpose.
```

Comment:
i can see an author wanting to name a modifier but namespace it differently; 'name' might be useful in authoring contexts, whereas `namespace` might be used in a parsing/resolving context.